app/game/[gameId]/view/page.tsx

```diff
@@ -9,7 +9,7 @@ export default async function GameHome({
   searchParams: { [key: string]: string | string[] | undefined }
 }) {
   const game = await getGame(params.gameId)
-  const gameCode = searchParams['gameCode']
+  const gameCode = (searchParams['gameCode'] as string).toUpperCase()
```

Comment:
Nit: This would throw an error if the gameCode is not specified. It is fatal but perhaps we want better error handling

Reply:
Oh fair. Do we want to just do like an alert("Please include game code!")?